smdebug/core/tfevent/util.py

```diff
@@ -27,12 +27,25 @@
     np.dtype(np.complex64): "DT_COMPLEX64",
     np.dtype(np.complex128): "DT_COMPLEX128",
     np.dtype(np.bool): "DT_BOOL",
+    np.dtype([("qint8", "i1")]): "DT_QINT8",
+    np.dtype([("quint8", "u1")]): "DT_QUINT8",
+    np.dtype([("qint16", "<i2")]): "DT_QINT16",
+    np.dtype([("quint16", "<u2")]): "DT_UINT16",
+    np.dtype([("qint32", "<i4")]): "DT_INT32",
 }
 
 
 def _get_proto_dtype(npdtype):
-    if npdtype.kind == "U":
+    if npdtype.kind == "U" or npdtype.kind == "O" or npdtype.kind == "S":
         return (False, "DT_STRING")
+    try:
+        from tensorflow.python import _pywrap_bfloat16
+
+        # TF Implements a Custom Numpy Datatype for Brain Floating Type
+        _np_bfloat16 = _pywrap_bfloat16.TF_bfloat16_type()
```

Comment:
There is no special ask for this, but the goal is to support every possible TF type.